courses/pyladies/branching/index.html

```diff
@@ -86,7 +86,7 @@
                         a pomocí <code>git add</code> a <code>git commit</code>
                         udělej novou revizi.
                         Pak koukni na <code>gitk --all</code>, jak to vypadá:
-                        <div><img src="{{ static('branch1.png') }}"></div>
+                        <div><img src="./static/branch1.png"></div>
```

Comment:
URLs should really be generated by a function; you can't assume a relationship between the file's URL and the lesson's URL. They might even be served by different servers if naucse becomes a dynamic site.